test/unit/voldemort/store/rocksdb/RocksdbStorageEngineTest.java

```diff
@@ -85,11 +83,9 @@ public void tearDown() throws Exception {
     @Override
     public void testTruncate() throws Exception {}
 
-    @Ignore
```

Comment:
Agreed.  My mistake.  I did my testing using a version of the unit test code that did not include those ignore/overrides.  I only saw that code when I patched it over to git.  In a brain dead move I just removed the Ignore.